libraries/rush-lib/src/logic/operations/ProjectLogWritable.ts

```diff
@@ -117,6 +165,13 @@ export class ProjectLogWritable extends TerminalWritable {
     // Both stderr and stdout get written to *.<phaseName>.log
     this._logWriter.write(chunk.text);
 
+    if (this._enableChunkedOutput) {
+      if (!this._chunkWriter) {
+        this._chunkWriter = FileWriter.open(this.logChunksPath);
+      }
+      this._chunkWriter.write(JSON.stringify(chunk) + '\n');
+    }
```

Comment:
I don't like creating files inside of the `write` callback. I'd rather pay the tax of creating this file up front and then letting it be empty.

Reply:
Also, there's a handler for `this._logWriter` being closed; we should have the same for `this._chunkWriter` for the same reason.

Reply:
Updated in [e531b9c](https://github.com/microsoft/rushstack/pull/4668/commits/e531b9cde8d88c03e0499d6f4a4ab714b9a8bf95)